pkg/deploy/registry/deploylog/rest.go

```diff
@@ -180,6 +186,31 @@ func (r *REST) Get(ctx kapi.Context, name string, opts runtime.Object) (runtime.
 	}, nil
 }
 
+// waitForExistingDeployment will use the timeout to wait for a deployment to appear.
+func (r *REST) waitForExistingDeployment(namespace, name string) (*kapi.ReplicationController, error) {
+	var (
+		target *kapi.ReplicationController
+		err    error
+	)
+
+	condition := func() (bool, error) {
```

Comment:
A paused deployment or no triggers don't mean we don't have an already running deployment. The command you posted is racy atm because we derive the version of the config in the server but we can make it bulletproof if we derive it in the client (we already send an individual `version` parameter to enable looking at older deployments).